vmdb/app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -67,4 +67,15 @@ def textual_containers
     end
     h
   end
+
+  def textual_container_nodes
+    count_of_nodes = 1 unless @record.container_node.nil?
```

Comment:
this should be changed to:
node = @record.container_node
    return nil if node.nil?
    label = ui_lookup(:table => "container_node")